app/templates/components/forms/form-field.hbs

```diff
@@ -6,6 +6,10 @@
         *
         {{ember-tooltip text='This field is required' delay=300 }}
       </span>
+    {{else if showOptionalMark}}
+      <span class='travis-form__optional-mark'>
+        (Optional)
+      </span>
```

Comment:
As I mentioned above, it'd be better to stick with one approach, mark either optional or required fields, not both.